libs/navigation/footer.js

```diff
@@ -0,0 +1,25 @@
+const blockConfig = {
+  name: 'global-footer',
+  targetEl: 'footer',
+  appendType: 'appendChild',
+};
+
+async function loadBlock() {
+  const { fedsGlobalConfig = {} } = window;
+  const miloLibs = fedsGlobalConfig.miloLibs || 'https://stage--milo--adobecom.hlx.page';
+
+  // Relative path can't be used, as the script will run on consumer's app
+  const { default: bootstrapBlock } = await import(`${miloLibs}/libs/navigation/bootstrapper.js`);
+  const { default: locales } = await import(`${miloLibs}/libs/utils/locales.js`);
+  const clientConfig = {
+    origin: miloLibs,
+    contentRoot: fedsGlobalConfig.contentRoot,
+    miloLibs: `${miloLibs}/libs`,
+    pathname: `/${fedsGlobalConfig.locale || ''}`,
+    locales: fedsGlobalConfig.locales || locales,
+  };
+
+  bootstrapBlock(clientConfig, blockConfig);
+}
+
+loadBlock();
```

Comment:
One thing that I wonder about, is if a consuming project should just import the module and pass params, rather than writing the configs on the window
```js
const {loadFooter} = import("https://adobe.com/libs/navigation/footer.js")
loadFooter({...params})
```

Reply:
I am not against it, but do we want our consumers to explicitly call the function to use the block?
Thoughts? @salonijain3 @narcis-radu @amitbikram